requirements/requirements.in

```diff
@@ -9,7 +9,7 @@ ipython==8.2.0
 setuptools==58.1.0
 
 # DB
-SQLAlchemy==1.4.36
+SQLAlchemy>=2.0
```

Comment:
As usual this and derived `requirements.txt` changes are only for the bazel dev environment. User dependency version requirements are set by the version constraints in `wheel_constants.bzl`